scrapers/zilean.py

```diff
@@ -172,21 +172,25 @@ async def process_stream(
                         self.metrics.record_skip("Missing season info")
                         return None
 
-                    if episodes := torrent_data.get("episodes"):
+                    if season not in seasons:
```

Comment:
FYI: I'm scraping all sort of the series and episode in here. But the actual issue is our DB corrupted with wrong episode annotation.